win/tty/termcap.c

```diff
@@ -737,52 +735,10 @@ static const short tmspc10[] = { /* from termcap */
 #endif
 #endif
 
-/* delay 50 ms */
+/* NLE: Don't delay ever. */
```

Comment:
Is this change somehow related to the bug fix? Or is it a drive-by perf win?

Reply:
Related in the sense that if one doesn't link against ncurses/terminfo/termcap by `#undef`'ing `TERMLIB`, there's symbols in `tty_delay_output` that the linker complains about.

Also we never called that function in the first place.